modules/subchannel/src/problems/TriSubChannel1PhaseProblem.C

```diff
@@ -469,12 +469,15 @@ TriSubChannel1PhaseProblem::computeBeta(unsigned int i_gap, unsigned int iz)
   auto Re = avg_massflux * avg_hD / avg_mu;
   // crossflow area between channels i,j (dz*gap_width)
   auto gap = _subchannel_mesh.getGapWidth(iz, i_gap);
+  auto dpgap = _tri_sch_mesh.getDuctToPinGap();
+  auto w = pin_diameter + dpgap;
```

Comment:
w? width? width of what? center to duct? so side channel next to the duct? why computed here when it's not relevant to CENTER channels